views/js/qtiItem/core/Item.js

```diff
@@ -83,6 +83,19 @@ define([
             }
             return this;
         },
+        getOutcomeDeclaration : function getOutcomeDeclaration(identifier){
+            var found;
+            _.each(this.outcomes, function (outcome) {
```

Comment:
`_.each()` is deprecated, please use `_.forEach()` instead.
You could also use `_.find()`